docs/src/hugo/content/introduction/competitors.md

```diff
@@ -2,30 +2,24 @@
 layout: "single"
 title: "Competitors"
 preamble: >
-  Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas eu commodo nisi. Etiam sagittis enim purus, id tristique ante tincidunt id. Nunc placerat velit neque. Integer finibus velit nec vestibulum elementum.
+  Nelson is in many ways a uniuqe system; the principles and methods used in its construction are not common in the operations domain. However, is is common for potential users to draw parrallels with other systems, and navigating the competitive space can often be difficult. The systems most commonly compared with Nelson are enumerated below a brief comparitive explanation is provided to help understand Nelson's value proposistion.
 contents:
 - Spinnaker
 - Collins
-#menu:
-#  main:
-#    parent: 'intro'
-#    identifier: intro-competitors
-#    url: /introduction/competitors.html
-#    weight: 3
+menu:
+  main:
+    parent: 'intro'
+    identifier: intro-competitors
+    url: /introduction/competitors.html
+    weight: 3
 ---
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas eu commodo nisi. Etiam sagittis enim purus, id tristique ante tincidunt id. Nunc placerat velit neque. Integer finibus velit nec vestibulum elementum. Mauris vel tempor velit. Nam sodales malesuada purus vel bibendum. Sed eu mauris sit amet nulla sodales imperdiet. Sed pulvinar iaculis magna at feugiat. Quisque luctus nec lectus vitae pharetra. Cras euismod pharetra augue ac consequat. Integer id euismod lectus, non cursus ligula. Nunc lorem dolor, condimentum nec nunc non, hendrerit ullamcorper tellus. Sed tortor nibh, mollis eget euismod ac, tempor a eros. Etiam feugiat odio ac interdum pellentesque. Nam dignissim, ipsum ut vehicula interdum, justo dui accumsan felis, in maximus diam justo tristique tellus.
-
 ## Spinnaker
```

Comment:
@okoye any chance you can read this modified section and provide a comment?

Reply:
@timperrett sure thing, I'll do that on my flight to Pittsburgh